rero/rero-ils
# indexing: update mef persons mapping

* Adds analyzer on biographical_information, preferred_name_for_person,
  authorized_access_point_representing_a_person and variant_name_for_person fields.

How to test:

* `script/setup`
* http://localhost:5000/admin/records/persons
* Find a term and check if the result is correct

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>
